docs/reference/api/drivers/LowPowerTimer.md

```diff
@@ -2,12 +2,14 @@
 
 <span class="images">![](https://os-doc-builder.test.mbed.com/docs/development/mbed-os-api-doxy/classmbed_1_1_low_power_timer.png)<span>LowPowerTimer class hierarchy</span></span>
 
-[Add description here.]
+LowPowerTimer inherits from the Timer Class. The timer in this case continues operating even in deep sleep mode. It relies on `lp_ticker`, which is the low power ticker.
+
+You can use the LowPowerTimer interface to create, start, stop and read a timer for measuring small times (between microseconds and seconds). You can independently create, start and stop any number of LowPowerTimer objects.
```

Comment:
Query: What's the benefit of independently creating, starting and stopping LowPowerTimer objects? What's the alternative?

Reply:
No special benefit or alternative, its essentially how timers work or are expected to work

Reply:
@deepikabhavnani - We also have Timer/Timeout classes. I think it would be good to capture hows LowPower versions different from generic Timer/Timeout classes, and when people should use this vs others.